thrall/app/lib/ThrallConfig.scala

```diff
@@ -34,6 +34,7 @@ class ThrallConfig(resources: GridConfigResources) extends CommonConfig(resource
   val writeAlias: String = string("es.index.aliases.write")
 
   val thumbnailBucket: String = string("s3.thumb.bucket")
+  val quarantineBucket: String = string("s3.quarantine.bucket")
```

Comment:
This isn't used by thrall so it would be best to not require it in the configuration.